src/org/omegat/MainClassLoader.java

```diff
@@ -0,0 +1,103 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+          with fuzzy matching, translation memory, keyword search,
+          glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2021 Hiroshi Miura
+               2018 Mordechai Meisels(Original licensed Apache-2.0)
```

Comment:
```suggestion
               2018 Mordechai Meisels (Original licensed Apache-2.0)
```

Reply:
Where exactly did this code come from?

Reply:
Come from here
https://github.com/update4j/update4j/blob/master/src/main/java/org/update4j/DynamicClassLoader.java
and

Reply:
You can see OpenJDK AppClassLoader (what getSystemClassLoader() returns) if want.
https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/jdk/internal/loader/ClassLoaders.java#L163

That is not child of `URLClassLoader` in Java9+, but behave like a `URLClassLoader`

Reply:
Thanks.